ui/src/store/resources.ts

```diff
@@ -0,0 +1,146 @@
+import { types, getEnv, flow, Instance } from "mobx-state-tree"
+import { Api } from "../api"
+import { ICategoryStore, Tag } from "./category"
+import { IKindStore, Kind } from "./kind"
+import { ICatalogStore, CatalogType } from "./catalog"
+import fuzzysort from "fuzzysort"
+
+const Catalog = types.model("Catalog", {
```

Comment:
Can we reuse the Catalog type here which is defined in the catalog store instead of having a new catalog type ?

Reply:
both model are different to each other